src/components/VerificationLog/WitnessResult.tsx

```diff
@@ -0,0 +1,135 @@
+
+const WitnessResult = ({
+  witnessResult,
+}: {
+  witnessResult: WitnessResultProps;
+}) => {
+  if (!witnessResult) return <Text>No Witness event detected</Text>;
+
+  const {
+    tx_hash,
+    witness_network,
+    witness_event_vh_matches,
+    merkle_proof_status,
+    actual_witness_event_verification_hash,
+    extra,
+    etherscan_error_message,
+  } = witnessResult;
+
+  const etherscanNetworkURL = `https://${witness_network}.etherscan.io/`;
+
+  const ConditionalIconListItem = ({
+    condition,
+    title,
+    content,
+    truncateContent,
+  }: {
+    condition: boolean;
+    title: string;
+    content: string;
+    truncateContent?: boolean;
+  }) => {
+    return (
+      <ListItem display="flex" alignItems="center" maxWidth="sm">
+        <ListIcon
+          as={condition ? CheckCircleIcon : CloseIcon}
+          color={condition ? "success" : "error"}
+          width="20px"
+        />
+        <Text color="grey">{title}</Text>
+        <Text isTruncated={truncateContent} marginLeft={1}>
+          {content}
+        </Text>
+      </ListItem>
+    );
+  };
+
+  const IconListItem = ({
+    icon,
+    title,
+    content,
+    truncateContent,
+  }: {
+    icon: any;
+    title: string;
+    content: string | React.ReactNode;
+    truncateContent?: boolean;
+  }) => {
+    return (
+      <ListItem display="flex" alignItems="center" maxWidth="sm">
+        <ListIcon as={icon} width="20px" />
+        <Text color="grey">{title}</Text>
+        <Text isTruncated={truncateContent} marginLeft={1}>
+          {content}
+        </Text>
+      </ListItem>
+    );
+  };
+
+  const Extra = () =>
+    extra && (
+      <>
+        <ListItem>{extra.domain_snapshot_genesis_hash}</ListItem>
+        <ListItem>{extra.merkle_root}</ListItem>
+        <ListItem>{extra.witness_event_verification_hash}</ListItem>
+      </>
+    );
+
+  const ScanNetwork = () => (
+    <Link href={etherscanNetworkURL} isExternal textDecoration="underline">
+      {witness_network}
+    </Link>
+  );
+
+  // ⛓ Witness transaction hash 0x17cb36....db79d7
+  // ✅ ⌚ Witness event hash has been verified on Goerli
+  // ✅ 🌿 Witness Merkle Proof is OK
+  // ❌ ⌚ Witness event can not be verified on Goerli
+  // ❌ Witness event verification hash doesn't match
+  // ❌ 🌿 Witness Merkle Proof is corrupted
+  return (
+    <List fontSize="sm" maxW="sm" spacing={3}>
+      {tx_hash && (
+        <IconListItem
+          icon={CompareArrowsIcon}
+          title="Transaction Hash"
+          content={tx_hash}
+          truncateContent
+        />
+      )}
+      <ConditionalIconListItem
+        condition={witness_event_vh_matches}
+        title="Verification Hash"
+        content={actual_witness_event_verification_hash}
+        truncateContent
+      />
+      <Extra />
+      {etherscan_error_message && (
+        <ListItem>{etherscan_error_message}</ListItem>
+      )}
+      <ConditionalIconListItem
+        condition={merkle_proof_status === "VALID"}
```

Comment:
This is not taking into account when `merkle_proof_status` is "DOMAIN_SNAPSHOT".

Reply:
added DOMAIN_SNAPSHOT to the condition, will handle rendering in another commit